e2e/imgbuild/imgbuild.go

```diff
@@ -2050,16 +2321,18 @@ func (c imgBuildTests) buildWithAuthTester(t *testing.T, withCustomAuthFile bool
 		}
 	})
 
-	dockerfile, err := e2e.WriteTempFile(tmpdir, "Dockerfile", fmt.Sprintf(
-		`
-FROM %s
-CMD /bin/true
-`,
-		privImgNoPrefix,
-	))
-	if err != nil {
-		t.Fatalf("while trying to generate test dockerfile: %v", err)
-	}
+	// The following is disabled because it is now subsumed under
```

Comment:
Just remove this code. Significant commented chunks inevitably get out of date eventually, and can then confuse.

You can keep the removal in a separate commit with a message explaining it if you want to record the context of the removal.